tests/cli/workflow/rnafusion/test_cli_rnafusion_compound_commands.py

```diff
@@ -208,3 +210,33 @@ def test_store_available(
 
     # THEN bundle added successfully and action set to None
     assert rnafusion_context.status_db.get_case_by_internal_id(case_id_success).action is None
+
+
+def test_start_available(
+    cli_runner: CliRunner,
+    rnafusion_context: CGConfig,
+    caplog: LogCaptureFixture,
+    mocker,
+    rnafusion_case_id: str,
+):
+    """Test to ensure all parts of compound start-available command are executed given ideal conditions
+    Test that start-available picks up eligible cases and does not pick up ineligible ones."""
+    caplog.set_level(logging.INFO)
+
+    # GIVEN CASE ID of sample where read counts pass threshold
+    case_id_success: str = rnafusion_case_id
+
+    # GIVEN a mocked config
+
+    # GIVEN decompression is not needed
+    mocker.patch.object(RnafusionAnalysisAPI, "resolve_decompression")
+    RnafusionAnalysisAPI.resolve_decompression.return_value = None
+
+    # WHEN running command
+    result = cli_runner.invoke(start_available, ["--dry-run"], obj=rnafusion_context)
+
+    # THEN command exits with 0
+    assert result.exit_code == EXIT_SUCCESS
+
+    # THEN it should successfully identify the one case eligible for auto-start
+    assert case_id_success in caplog.text
```

Comment:
I sort of like having a bit more context since for someone who does not know the flow, maybe the case_id is logged for failures as well. But feel free to disregard, you know the flow better

Reply:
I agree with you. Tbh, the tests for compound commands need some love. I will open an issue and do this in a separate PR

Reply:
open issue: https://github.com/Clinical-Genomics/cg/issues/2471